justfile

```diff
@@ -1,145 +1,145 @@
 set dotenv-load := false
 
-# Show all available recipes
-default:
-  @just --list --unsorted
+# Meaning of Just prefixes:
+# @ - Quiet recipes (https://github.com/casey/just#quiet-recipes)
+# _ - Private recipes (https://github.com/casey/just#private-recipes)
 
+# The monorepo uses `PROD` instead of `IS_PROD`.
 IS_PROD := env_var_or_default("IS_PROD", "")
-DOCKER_FILES := "--file=docker-compose.yml" + (
-    if IS_PROD != "true" {" --file=docker-compose.override.yml"} else {""}
-)
-SERVICE := env_var_or_default("SERVICE", "scheduler")
```

Comment:
It looks like we're getting rid of the `SERVICE` default variable for some recipes (namely `exec` and `run`). Do you mind updating any references to `just exec` or `just run` in the `README.md` to include the `scheduler` service? E.g. `just run scheduler airflow dags list`

Reply:
Updated in 24a858e.